exporter/elasticsearchexporter/logs_exporter.go

```diff
@@ -69,8 +60,6 @@ func newLogsExporter(logger *zap.Logger, cfg *Config) (*elasticsearchLogsExporte
 
 		index:          indexStr,
 		dynamicIndex:   cfg.LogsDynamicIndex.Enabled,
-		maxAttempts:    maxAttempts,
-		retryOnStatus:  cfg.Retry.RetryOnStatus,
```

Comment:
Aren't these breaking changes for the user?

Reply:
No, there is no functional change here. It is no longer storing maxAttempts and retryOnStatus separately within the exporter because they are already passed into elasticsearch client (for request level retries) and bulk indexer (for document level retries) in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32359/files#diff-a09f63572f3eb4cbff2f4ef096a2cd5c71025e6e3c882617d466f57120aeefd8R36

Reply:
Yep, I see it now.  Thanks!